project-base/storefront/components/Blocks/SimpleNavigation/SimpleNavigationListItem.tsx

```diff
@@ -21,11 +21,28 @@ export const SimpleNavigationListItem: FC<SimpleNavigationListItemProps> = ({
     const itemImage = 'mainImage' in listedItem ? listedItem.mainImage : null;
     const href = getStringWithoutTrailingSlash(listedItem.slug) + '/';
 
+    const getLinkType = (type: ListedItemPropTypeTypename | undefined): PageType => {
+        switch (type) {
+            case 'ArticleSite':
+                return 'article';
+            case 'BlogArticle':
+                return 'blogArticle';
+            case 'Brand':
+                return 'brand';
+            case 'Category':
+                return 'category';
+            default:
+                throw new Error('Link type ' + type + ' could not be resolved.');
```

Comment:
This seems harsh. I am not sure, maybe this is a business requirement and something that would make the developer notice it during development, but this essentially kills the entire FE. What I think would be better is send a message to sentry and then not display that link

Reply:
After discussion we have decided to keep it as it is

Reply:
Just to add, I will discuss this with other FE developers and find a general solution we will apply to the entire codebase.